### ISSUE TITLE
Day headers cause WGAC audit fail due to missing <thead> in Firefox

### ISSUE THREAD

**Issue by danderson421** [fullcalendar/fullcalendar#6388]
### Reduced Test Case
https://codepen.io/danderson421/pen/XWMQBNZ?editors=001

### Bug Description

Describe how to recreate the bug. - use the calendar in Month view
What do you expect to happen? - the day name headings would be in thead and th tags
What happens instead? the day name headings are in td tags and missing thead

`<table class="fc-scrollgrid "><thead><tr class="fc-scrollgrid-section fc-scrollgrid-section-header ">`
should be
`<table class="fc-scrollgrid "><thead><th class="fc-scrollgrid-section fc-scrollgrid-section-header ">`

AND th elements should be in thead here too:

```
<table class="fc-col-header ">
<colgroup></colgroup>
<tbody>
<tr>
<th class="fc-col-header-cell fc-day fc-day-sun">
<th class="fc-col-header-cell fc-day fc-day-mon">
<th class="fc-col-header-cell fc-day fc-day-tue">
<th class="fc-col-header-cell fc-day fc-day-wed">
<th class="fc-col-header-cell fc-day fc-day-thu">
<th class="fc-col-header-cell fc-day fc-day-fri">
<th class="fc-col-header-cell fc-day fc-day-sat">
</tr></tbody></table>
```

monthview fails WGAC accessibility audit because of this.

**Comment by acerix** [comment]
The issue seems specific to Firefox, I get the expected `<thead>` in  Chrome. I believe it may be intentional to fix the bug mentioned here:

https://github.com/fullcalendar/fullcalendar/blob/a447a7976e6793a038a3c28d533d6aa53801125c/packages/common/src/scrollgrid/SimpleScrollGrid.tsx#L86

![Screenshot from 2021-06-21 18-16-57](https://user-images.githubusercontent.com/379534/122834714-14875f00-d2bd-11eb-9f17-161f0efed9d9.png)

**Comment by danderson421** [comment]
This happens in Chrome as well. Essentially all tables should have <thead> and all <thead> elements should have <tr><th> elements. There are two tables when day headers are on and the month view is used; both tables have minor issues. The fact that <thead> is there, is not quite good enough. The <thead> within the fc-scrollgrid table does not have a <th>, it has a <td>. The fc-col-header table does not have <thead>.

**Comment by arshaw** [comment]
I've created a new ticket with many more implementation details:
https://github.com/fullcalendar/fullcalendar/issues/6526
&uarr;
Could you please visit the link, **give a** 👍  **to vote** ([?](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments)), press the 🔈**Subscribe** button to receive updates?

I will be closing this ticket.
